pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -34,7 +34,7 @@ type InternetCheckerSpec struct {
 type ClusterSpec struct {
 	// ResourceID is the Azure resourceId of the cluster
 	ResourceID      string              `json:"resourceId,omitempty"`
-	ACRName         string              `json:"acrName,omitempty"`
+	ACRDomain       string              `json:"acrDomain,omitempty"`
```

Comment:
How does this behave on upgrade?  Does k8s complain when the CRD is updated and a field goes away?  Is it actually feasible to do this or do we need one release with ACRName and ACRDomain in parallel in the Cluster object?  Could the old operator briefly do bad things between the CRD being updated and the operator being updated?

Reply:
Let's have both for one release.

Reply:
TODO/and VSTS story would be good here.

Reply:
@nilsanderselde did you tested this? Like create old cluster, and try run "admin patch" from your customer branch? to make sure we don't have unknown unknowns?

Reply:
Yes, ran it locally on master, created cluster, ran it on this branch, patched cluster - worked as intended.